content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/saml-sso.md

```diff
@@ -45,6 +39,141 @@ We don’t synchronize user datastores, so make sure that all users:
+    - Under **Basic SAML Configuration**, enter:
+       - **Identifier** (Entity ID): `https://api.cobalt.io/users/saml/metadata`
+       - **Reply URL** (Assertion Consumer Service URL): `https://api.cobalt.io/users/saml/auth`
+       - **Sign on URL**: Leave this field blank for IdP-initiated SAML.
+       - **Relay State**: Leave this field blank.
+       - **Logout URL**: leave this field blank.
+    - Under **User Attributes & Claims**, add custom attribute mappings to your SAML token attributes configuration.
+       - `givenname`: `user.givenname`
+       - `surname`: `user.surname`
+       - `emailaddress`: `user.mail`
+       - `name`: `user.userprincipalname`
+       - `Unique User Identifier`: `user.userprincipalname`
+    - The Cobalt app expects the following attributes to be passed in the SAML response:
+       <table style="border: 1px solid #E0E2E6; padding: 5px;"><thead><tr><th style="border: 1px solid #E0E2E6; padding: 5px;">Name</th><th style="border: 1px solid #E0E2E6; padding: 5px;">Source Attribute</th></tr></thead><tbody><tr><td style="border: 1px solid #E0E2E6; padding: 5px;"><code>Mail</code></td><td style="border: 1px solid #E0E2E6; padding: 5px;"><code>user.mail</code></td></tr><tr><td style="border: 1px solid #E0E2E6; padding: 5px;"><code>Othermail</code></td><td style="border: 1px solid #E0E2E6; padding: 5px;"><code>user.othermail</code></td></tr></tbody></table>
+    - Under **SAML Signing Certificate**, download **Certificate (Base 64)**.
+    - Under **Set up [Your App]**, copy the required values.
+1. [Assign users](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-assign-users#assign-a-user-account-to-an-enterprise-application) to the application.
+1. Configure SAML settings in the Cobalt app in **Settings** > **Identity & Access** > **SAML 2.0**.
+    - For these parameters, enter values from Azure AD:
+      - **IdP Issuer URL**: Azure AD Identifier
+      - **IdP Target URL**: Login URL
+      - **IdP Certificate**: Certificate (Base64)
+    - Save the **IdP Relay State** value that appears in red.
+1. In Azure AD, under **Basic SAML Configuration**, paste the **IdP Relay State** value into **Relay State**.
+1. [Test your configuration](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso#test-single-sign-on).
+
+### Duo
+
+To set up SAML SSO with Duo, read their [documentation](https://duo.com/docs/sso-generic).
+
+| For This Parameter in Cobalt | Enter This Value from Duo |
+|---|---|
+| IdP Issuer URL | Entity ID |
+| IdP Target URL | Single Sign-On URL |
+| IdP Certificate | Certificate |
+
+<br>
+
+| For This Parameter in Duo | Enter This Value from Cobalt |
+|---|---|
+| Entity ID | `https://api.cobalt.io/users/saml/metadata` |
+| Assertion Consumer Service (ACS) URL | `https://api.cobalt.io/users/saml/auth` |
+| Default Relay State | Your `IdP Relay State` from the Cobalt app |
```

Comment:
Should we just say IdP Relay State instead of IdP Relay State "from the Cobalt app" since this is within the headline?